src/land/afwa_snod2ioda.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+#
+# (C) Copyright 2020 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+import pygrib
+import time, os, sys
+import argparse
+import netCDF4 as nc
+import numpy as np
+from datetime import datetime, timedelta
+from pathlib import Path
+
+IODA_CONV_PATH = Path(__file__).parent/"@SCRIPT_LIB_PATH@"
+if not IODA_CONV_PATH.is_dir():
+    IODA_CONV_PATH = Path(__file__).parent/'..'/'lib-python'
+sys.path.append(str(IODA_CONV_PATH.resolve()))
+
+import ioda_conv_ncio as iconv
+from orddicts import DefaultOrderedDict
+
+vName = {
+    'A': "afwa_snow_depth",
+}
+
+locationKeyList = [
+    ("latitude", "float"),
+    ("longitude", "float"),
+    ("datetime", "string")
+]
+
+AttrData = {}
+
+
+class AFWA(object):
+
+    def __init__(self, filename, mask, writer):
+        self.filename = filename
+        self.mask = mask
+        self.data = DefaultOrderedDict(lambda: DefaultOrderedDict(dict))
+        self.writer = writer
+        self._read()
+
+    def _read(self):
+        data = pygrib.open(self.filename)
+        lat, lon = data[1].latlons()
+        lons = lon[:].ravel()
+        lats = lat[:].ravel()
+        vals = data[1].values[:].ravel()
+        # defined errors and qc
+        errs = 0.15*vals
+
+        if self.mask == "maskout":
+            mask = np.logical_not(vals.mask)
+            vals = vals[mask]
+            lons = lons[mask]
+            lats = lats[mask]
+            errs = errs[mask]
+        # get global attributes
+        start_datetime = data[1].analDate
+        base_datetime = start_datetime.isoformat() + "Z"
+        # grbs.close()
```

Comment:
Why is this commented out? either remove it or un-comment it

Reply:
As it does not need to be closed so that I removed as suggested.

Reply:
As in many languages, the program may "survive" just fine at normal exit to close any open files, but I always think it is a good idea to close files explicitly as early in a program as possible before various computations/calculations or writing of other files.  That way is something does go wrong, memory or other resources towards open files is reduced.

Reply:
Based on Greg's comments and your concern here, I added data.close() back to close this opened grib file after the code finished reading @CoryMartin-NOAA @gthompsnJCSDA